src/neuroglancer/layer_panel.ts

```diff
@@ -28,6 +28,7 @@ import {getDropEffect, preventDrag, setDropEffect} from 'neuroglancer/util/drag_
 import {float32ToString} from 'neuroglancer/util/float32_to_string';
 import {makeCloseButton} from 'neuroglancer/widget/close_button';
 import {PositionWidget} from 'neuroglancer/widget/position_widget';
+import {AnnotationUserLayer} from './annotation/user_layer';
```

Comment:
Change to `'neuroglancer/annotation/user_layer'` so that the import sorting works correctly.